app/zadanie02.js

```diff
@@ -1 +1,18 @@
-//Twój kod
\ No newline at end of file
+//Twój kod
+let arr = {
```

Comment:
Jedna mała podpowiedź - jak coś jest stałe (nie zamierzamy tego zmieniać w trakcie trwania programu - jak tu) to oznaczamy to jako `const`.

Reply:
dzięki!